runtime/compiler/env/VMJ9.h

```diff
@@ -1328,6 +1338,8 @@ class TR_J9SharedCacheVM : public TR_J9VM
    virtual bool               needsContiguousCodeAndDataCacheAllocation()     { return true; }
    virtual bool               needRelocatableTarget()                          { return true; }
    virtual bool               isStable(int cpIndex, TR_ResolvedMethod *owningMethod, TR::Compilation *comp) { return false; }
+   virtual bool               isForceInline(TR_ResolvedMethod *method) { return false; }
```

Comment:
Is it easy to enable this functionality for AOT compiles ? I assume the above code disables it. @dsouzai fyi

Reply:
Yeah it looks like it's currently disabled in AOT. The default implementation is:
```
bool
TR_J9VMBase::isForceInline(TR_ResolvedMethod *method)
   {
   return jitIsMethodTaggedWithForceInline(vmThread(),
                                           (J9Method*)method->getPersistentIdentifier());
   }
```
which is
https://github.com/eclipse-openj9/openj9/blob/9af2e37a0d993b70055f972c02e6bb3c61a0a9b3/runtime/jit_vm/cthelpers.cpp#L219-L223
which calls
https://github.com/eclipse-openj9/openj9/blob/9af2e37a0d993b70055f972c02e6bb3c61a0a9b3/runtime/util/annhelp.c#L230-L252
which looks like a ROMMethod/ROMConstantPool query. If that's the case then we don't need to do anything special for AOT because if the VM loads a ROMClass from the SCC in the second run, then they will have already ensured it is the same ROMClass as in the first run. 

@pshipton could you confirm that the code that checks the annotation above only deals with ROM* structures?

Reply:
Also a question to @tajila

Reply:
Yes we are only looking at the ROM classes for annotation data. That being said, if HCR occurs (which can modify annotations) we generate a new romClass, so in those cases you'd need to be aware of potential changes.

Reply:
Regarding HCR, if the class was redefined before the load then we would fail the AOT load because the class chain wouldn't match. If it happened after the load, either a runtime assumption or just the normal runtime semantics of HCR would ensure correctness.